config/database.yml

```diff
@@ -24,10 +24,16 @@ default: &default
 development:
   <<: *default
   database: smogmapper_development
+  username: <%= ENV['SMOGMAPPER_DB_USERNAME'] %>
+  password: <%= ENV['SMOGMAPPER_DB_PASSWORD'] %>
+  host: <%= ENV['SMOGMAPPER_DB_HOST'] %>
```

Comment:
Is this required? I mean it will force people, who do not use docker, to define these things, right?